torch/fx/passes/backends/nvfuser.py

```diff
@@ -266,15 +266,16 @@ def compile(self, graph_module: GraphModule) -> GraphModule:
             logging.debug("partitioner_cache hit!")
             fused_graph_module = self.partitioner_cache[graph_module]
         else:
-            partitioner = CapabilityBasedPartitioner(graph_module, self.supported_ops)
+            partitioner = CapabilityBasedPartitioner(
```

Comment:
If we are cleaning/tinkering the fx execution, maybe we also want to cherry-pick this guy https://github.com/pytorch/pytorch/pull/81820/files#diff-8423e87cb2c44e61ff9701c15b9f55a6f9614066dcf1e0ce8c35d571db6d0f15R207-R208